src/test/java/com/sleekbyte/tailor/functional/FormatTest.java

```diff
@@ -89,100 +97,93 @@ public void testXcodeFormat() throws IOException {
     public void testJSONFormat() throws IOException {
         Format format = Format.JSON;
 
-        String[] command = new String[] {
+        final String[] command = new String[] {
             "--format", format.getName(),
             "--no-color",
             "--only=upper-camel-case",
             inputFile.getPath()
         };
 
-        addAllExpectedMsgs(format);
-        expectedMessages.addAll(Arrays.asList(getJSONSummary(1, 0, 0, 22).split(NEWLINE_REGEX)));
+        List<Map<String, Object>> violations = new ArrayList<>();
+        for (ViolationMessage msg : getExpectedMsgs()) {
+            Map<String, Object> violation = new HashMap<>();
+            Map<String, Object> location = new HashMap<>();
+            location.put(Messages.LINE_KEY, msg.getLineNumber());
+            location.put(Messages.COLUMN_KEY, msg.getColumnNumber());
+            violation.put(Messages.LOCATION_KEY, location);
+            violation.put(Messages.SEVERITY_KEY, msg.getSeverity().toString());
+            violation.put(Messages.RULE_KEY, Rules.UPPER_CAMEL_CASE.getName());
+            violation.put(Messages.MESSAGE_KEY, msg.getMessage());
+            violations.add(violation);
+        }
+        Map<String, Object> file = new HashMap<>();
```

Comment:
Can you break this block of code into smaller sections by adding newlines in between - just to help with readability.